files/es/web/javascript/reference/errors/missing_semicolon_before_statement/index.md

```diff
@@ -8,7 +8,7 @@ slug: Web/JavaScript/Reference/Errors/Missing_semicolon_before_statement
 ## Mensaje
 
 ```
-Error de Sintaxis: Punto y coma ; faltante antes de la declaración
+SyntaxError: missing ; before statement
```

Comment:
Now all the messages here are in English lang, I wonder if that should be the way to go or was okay as they were in Spanish.
@Graywolf9

Reply:
These were manual changes I had made, admittedly -- in browsers, the error messages are not translated when selecting a different language (I tested this in Chrome and Firefox), so translating the error messages has a negative SEO impact.

Reply:
Neither Firefox nor Brave (Chromium) shows that message, but thats a content suggestion for en-US first, also I remember some errors that indeed are shown in Spanish so maybe thats a thing that should be manually checked.

Thank you for everthing.

Firefox:
```
Uncaught SyntaxError: unexpected token: identifier
```

Brave:
```
Uncaught SyntaxError: Unexpected identifier 's'
```